multimedia/stremio/doinst.sh

```diff
@@ -6,4 +6,9 @@ if [ -e usr/share/icons/hicolor/icon-theme.cache ]; then
     /usr/bin/gtk-update-icon-cache -f usr/share/icons/hicolor >/dev/null 2>&1
   fi
 fi
-ln -sf "$(which node)" "/usr/share/stremio/node"
+which_node=$(which node 2>/dev/null)
```

Comment:
Shouldn't this be without the leading `/`? The error output we are seeing in the build output is because the package is being installed into a chroot with a `--root` flag.

Reply:
thanks for feedback, when the script is used without chroot it looks for where node it installed "At installed time" it makes it work if the user have installed node at any path. but with chroot and without node in there. What wont work. now it will default to /use/share/ anyway. Yes that should be with a leading / its forcing the link there.